html/elements/script.json

```diff
@@ -558,13 +558,16 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "61"
+                  "version_added": "61",
+                  "notes": "Module scripts without the <code>async</code> attribute do not load when the page is served as XHTML (<code>application/xhtml+xml</code>). See <a href='https://crbug.com/717643'>bug 717643</a>."
                 },
                 "chrome_android": {
-                  "version_added": "61"
+                  "version_added": "61",
+                  "notes": "Module scripts without the <code>async</code> attribute do not load when the page is served as XHTML (<code>application/xhtml+xml</code>). See <a href='https://crbug.com/717643'>bug 717643</a>."
                 },
                 "edge": {
-                  "version_added": "16"
+                  "version_added": "16",
```

Comment:
Speaking of testing, I'd be good to double-check this point, too. It's possible the behavior of EdgeHTML (Edge <79) is different from Chromium-based Edge (Edge ≥79)

Reply:
> I have access to some testing tools. I'm happy to test this, if you can point me toward a simple test case.

Oh sorry for not getting back to you sooner, I'll probably just link the test case that I had sent to Chromium previously.
I should have it sent within a few minutes (I might take a few hours though).

Reply:
Upon testing: EdgeHTML fully supported this and Chromium Edge has the Chrome behavior. So we'll need an array of support statements here:

1. `"version_added": "79"` with the same note as Chrome
2. `"version_added": "16"` and `"version_removed": "79"` (no note)